test/Spec.hs

```diff
@@ -23,10 +23,10 @@ gather (hout, herr, ph) = do
   pure (code, cont)
 
 files :: [FilePath]
-files = map ("examples/"++) $
+files = fmap ("examples/"++)
   -- This file does check, but it often fails when limited to reasonable time
-  -- [ "chinese.ftl"
-  [ "fuerst.ftl"
+  [ "chinese.ftl"
```

Comment:
I took "chinese.ftl" out because of the comment above. I should have clarified that this usually happens on travis, so you might not have detected it on your local build.

Reply:
I implemented this proposed change and also set the time limit back to 25.